config.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/gravwell/gcfg"
+	"github.com/keybase/managed-bots/base"
+)
+
+const version = "1.0.0"
+
+type Options struct {
+	*base.Options
+	um             map[string]string
+	HTTPPrefix     string
+	WebhookSecret  string
+	GiteaURL       string
+	DirectMessages bool
+}
+
+type userMaps varconfig
+
+type varconfig struct {
```

Comment:
`varconfig` isn't the most descriptive name, is there something that would be more explanatory?